jdaviz/utils.py

```diff
@@ -116,15 +116,11 @@ def enable_hot_reloading():
             ' Please install with `pip install watchdog`'))
 
 
-def bqplot_clear_context():
-    """Clears hidden context of `bqplot.pyplot` module
-    to mimic matplotlib ``clf()``. This is necessary when
-    we draw multiple plots across different plugins.
+def bqplot_clear_figure(fig):
```

Comment:
Maybe @maartenbreddels know of a better way but I couldn't find it in `bqplot` doc, example, or quick code-dive.